bundle-workflow/src/test_workflow/put_file_into_S3.py

```diff
@@ -0,0 +1,361 @@
+                                   <tr>
+                                   <td rowspan="2"></td>
+                                   <th colspan="3" scope="colgroup">operationsCount</th>
+                                   <th colspan="3" scope="colgroup">requestsPerSecond</th>
+                                   <th colspan="4" scope="colgroup">latencyMillis</th>
+                                   </tr>
+                                   <tr>
+                                    <th scope="col">opCount</th>
+                                    <th scope="col">opErrorCount</th>
+                                    <th scope="col">opErrorRate</th>
+                                    <th scope="col">p0</th>
+                                    <th scope="col">p50</th>
+                                    <th scope="col">p100</th>
+                                    <th scope="col">p50</th>
+                                    <th scope="col">p90</th>
+                                    <th scope="col">p99</th>
+                                    <th scope="col">p100</th>
+                                   </tr>
+                                   <tr>
+                                       <th scope="row">auto-date-histogram</th>
+                    """
+                it = testResults_Operations('auto-date-histogram')
+                for i in range(1, 11):
+                    text = text + "<td>" + str(it.get(i)) + "</td>"
+                text = text + "</tr>" + \
+                    """
+                                   <tr>
+                                       <th scope="row">date-histogram-with-tz</th>
+                    """
+                it = testResults_Operations('date-histogram-with-tz')
+                for i in range(1, 11):
+                    text = text + "<td>" + str(it.get(i)) + "</td>"
+                text = text + "</tr>" + \
+                    """
+                                   <tr>
+                                       <th scope="row">date-histogram</th>
+                    """
+                it = testResults_Operations('date-histogram')
+                for i in range(1, 11):
+                    text = text + "<td>" + str(it.get(i)) + "</td>"
+                text = text + "</tr>" + \
+                    """
+                                    <tr>
+                                       <th scope="row">auto-date-histogram-with-tz</th>
+                    """
+                it = testResults_Operations('auto-date-histogram-with-tz')
+                for i in range(1, 11):
+                    text = text + "<td>" + str(it.get(i)) + "</td>"
+                text = text + "</tr>" + \
+                    """
+                                   <tr>
+                                       <th scope="row">index-append</th>
+                    """
+                it = testResults_Operations('index-append')
+                for i in range(1, 11):
```

Comment:
Instead iterate over your data, not a constant value.

 If the number of elements changes consider how many lines of code need to be updated, yikes is how many!